docs/tutorials/working_with_the_ensemble.ipynb

```diff
@@ -19,7 +19,7 @@
   },
```

Comment:
<ol><li>Reader could be confused by the fact that object frame has 0 entries and source frame has 1000 entries</li><li>Cell output and text do not match: 1000 vs 2000 rows, 36.1KB vs 54.7 KBs</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tape/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Yeah the previous numbers are from when we were reading from parquet as the data driver for this demo.</p>

Reply:
<p>Updated with the correct number of entries and removed the exact memory information.</p><p> </p><p>Also generated a slightly more interesting example for the Source and Object tables.</p>